src/scheduler.rs

```diff
@@ -132,26 +136,25 @@ async fn fetch(ctx: &Context, connection: &mut Imap) {
     match ctx.get_config(Config::ConfiguredInboxFolder).await {
         Ok(Some(watch_folder)) => {
             if let Err(err) = connection.prepare(ctx).await {
-                error_network!(ctx, "{}", err);
+                warn!(ctx, "Could not connect: {}", err);
                 return;
             }
 
             // fetch
             if let Err(err) = connection.fetch(ctx, &watch_folder).await {
-                connection.trigger_reconnect();
+                connection.trigger_reconnect(ctx).await;
                 warn!(ctx, "{:#}", err);
             }
         }
         Ok(None) => {
-            warn!(ctx, "Can not fetch inbox folder, not set");
-            connection.fake_idle(ctx, None).await;
```

Comment:
fetch() is called inbetween jobs if there are many jobs in the pipeline, and we don't want to enter fake_idle just because the inbox folder is not set